src/common/utils/get-startable-stoppable.ts

```diff
@@ -3,58 +3,85 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
-type Stopper = () => Promise<void> | void;
-type Starter = () => Promise<Stopper> | Stopper;
+export type Stopper = () => Promise<void> | void;
+export type Starter = () => Promise<Stopper> | Stopper;
```

Comment:
Do these internal types need to be exported?

Reply:
They are not internal, they form part of the exported function signature